### DISCUSSION TITLE
How can I ensure that LazyVim does not override my keys on updates?

### DISCUSSION THREAD

**Discussion by danielo515** [LazyVim/LazyVim#1496] [Q&A]
Hello.
I am someone that uses the editor for working, so the less surprises the better, and once I have it configured to my liking I prefer to stick with that.

However, in the rare ocassions that I decide to update LazyVim I always find some bad surprise.
The last one was that my keymap for searching `<c-f>` has now been overriden by scrolling.
How can I configure lazyvim so it never overrides my keys configurations? It is very hard to figure out from where the override is coming from, and I don't want to write a super long list of 'unmaps' in my config just to prevent this.

I think it will be nice to detect and respect existing user configurations.
Cheers

**Comment by dpetka2001** [comment]
Could you plz take a look at #1485? Maybe it's related to your problem. If that's the case, you would have to wait for the next LazyVim release, as that will provide a fix. If you're facing a different problem, please provide more details.

**Reply by danielo515** [reply]
The keymaps points to:
```
11:55:40 msg_show   verbose nmap <c-f> n  <C-F>       * <Lua 237: ~/.local/share/Lazy/lazy/LazyVim/lua/lazyvim/plugins/ui.lua:270>
                 Scroll forward
	Last set from Lua
```

So, indeed, it seems as exactly the same problem as #1485

**Reply by dpetka2001** [reply]
Please wait to see if it will work correctly with the next Lazyvim release and the fix that will provide. Or if you'd rather follow the latest development version of `Lazyvim` and `lazy.nvim` you could take a look at official [docs](https://www.lazyvim.org/configuration/lazy.nvim) for how to do that.
